WordPress/Classes/ViewRelated/Post/LoadMoreCounter.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+/// Temporary counter to help investigate an old issue with load more analytics being spammed.
+/// See https://github.com/wordpress-mobile/WordPress-iOS/issues/6819
+class LoadMoreCounter {
+
+    var count: Int = 0
```

Comment:
In the spirit of the comment from my initializer, this _could_ be private if you don't want it to be modified externally.

If you'd like it to be "read only", you could change it to the following:
```
private(set) var count: Int = 0
```

Reply:
Another good call.